agent/engine/common_unix_integ_testutil.go

```diff
@@ -294,23 +316,47 @@ func verifyFirelensDataDir(t *testing.T, firelensDataDir, firelensContainerUser
 }
 
 // verifyFirelensLogs verifies that the Firelens container sent app logs to a local file as expected
-func verifyFirelensLogs(t *testing.T, firelensTask *apitask.Task, tmpDir string) {
-	logFilePath := filepath.Join(tmpDir, "app.log")
+func verifyFirelensLogs(t *testing.T, firelensTask *apitask.Task, firelensType, tmpDir string) {
+	// Define the expected log file path based on the firelens type
+	var logFilePath string
+	var logFilePattern string
+	if firelensType == firelens.FirelensConfigTypeFluentbit {
+		// For Fluentbit, we expect a specific file name
+		logFilePath = filepath.Join(tmpDir, "app.log")
+		logFilePattern = logFilePath // Exact file path
+	} else {
+		// For Fluentd, we'll look for any .log file, since the file name is not deterministic
+		logFilePattern = filepath.Join(tmpDir, "*.log")
+	}
 
-	// Wait for the log file to be created and populated
+	// Wait for the log file to appear and have content
 	var logContent []byte
 	var err error
-	for i := 0; i < 30; i++ {
-		logContent, err = os.ReadFile(logFilePath)
-		if err == nil && len(logContent) > 0 {
+	var found bool
+	for i := 0; i < 60; i++ {
```

Comment:
nit: Can we store `60` somewhere with a meaningful name, maybe something like `maxCheckLogFileTimes`?

Reply:
Sure